ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -162,6 +172,11 @@ class TabBarItemView: UIControl {
         static let badgeBorderWidth: CGFloat = 2
         static let badgeHorizontalPadding: CGFloat = 10
         static let badgeCorderRadii: CGFloat = 10
+        static let unreadDotPortraitOffsetX: CGFloat = 6.0
+        static let unreadDotOffsetX: CGFloat = 4.0
+        static let unreadDotOffsetY: CGFloat = 20.0
+        static let unreadDotSize: CGFloat = 8.0
+        static let unreadDotColor: UIColor = UIColor(colorValue: GlobalTokens.sharedColors(.red, .primary))
```

Comment:
curious is the badge color different from unreaddot color?

Reply:
Until we merge this change to fluent 2 colors, the badge color is correct enough. I will remove the unreadDotColor since we're not using it any more.